packages/@cdktf/cli-core/src/lib/models/deploy-machine.ts

```diff
@@ -293,7 +301,9 @@ export function createAndStartDestroyService(options: {
   });
 
   options.varFiles?.forEach((v) => {
-    args.push(`-var-file='${v}'`);
+    args.push(
+      os.platform() === "win32" ? `-var-file=${v}` : `-var-file='${v}'`
+    );
```

Comment:
should we add this to deploy as well?

Reply:
lol, I never did the escaping in deploy so the bug was not there for this reason :D